docs/docs/configuration/overview.md

```diff
@@ -5,6 +5,8 @@ title: Overview
 
 `oauth2-proxy` can be configured via [config file](#config-file), [command line options](#command-line-options) or [environment variables](#environment-variables).
 
+### Cookie Secret
```

Comment:
I like the idea, maybe `Generating a Cookie Secret`?

Could you perhaps also add this to the v7/v6 versioned docs as well? I think this is worth back porting

Reply:
Sure.

Reply:
@JoelSpeed done.